doc/examples.md

```diff
@@ -4,7 +4,7 @@ Here you can find a bunch of simple examples for using ct, with some explanation
 
 ## Users and groups
 
-```yaml
+```container-linux-config
```

Comment:
Actually, I'm going to change this to `yaml container-linux-config`. That way, both GitHub and our site render it properly.